### DISCUSSION TITLE
Listing of Members

### DISCUSSION THREAD

**Discussion by molschimke2** [outline/outline#6642] [Ideas]
A viewer can read the user database and user group definitions via the preferences:

<img width="233" alt="image" src="https://github.com/outline/outline/assets/81193827/c5bff4bd-aa12-44f8-ae8b-7ef1052006da">

This is a privacy issue as the full names of the users are presented. A viewer (non-admin) should not see the users and groups.

**Comment by tommoor** [comment]
Outline is designed as a team knowledge base, there is something of a built in assumption that once logged in it's a trusted environment. I think we can introduce some options for things like this though

**Reply by stepankoci** [reply]
If we are in a trusted environment, then why do we set user groups and collection rights?

From our point of view, it would be really convenient to be able to disable access to members and groups.
Please consider this option.
Thank you very much

**Reply by molschimke2** [reply]
> Outline is designed as a team knowledge base, there is something of a built in assumption that once logged in it's a trusted environment. I think we can introduce some options for things like this though

By exposing the list of users, you are limiting Outline to this use case (team knowledge base). We use it as a non-public documentation platform, where users get read-only rights on specific collections. For me, this sounds not unreasonable or edgy but having the list of users exposed means that we might have to either fix the issue in a local fork or use another Wiki platform.

**Reply by stepankoci** [reply]
We are in the same situation and think about it the same way.

**Comment by molschimke2** [comment]
We use it for publishing documentation. and depending on their levels or partner agreements users have access to more or less documentation. Thats where the users and groups come into play.

**Reply by molschimke2** [reply]
It should be possible to hide members and groups from non-admins IMHO.

**Comment by SilasPeters** [comment]
I agree with that once logged in you are in a trusted environment and should be able to see who else is a member, but privacy is still worth a lot. I trust my fellow students, but they don't have to know my email.

I believe this is a privacy principle: why not just hide the list, or give users a stripped down version of their email as their standard username?

**Reply by molschimke2** [reply]
Hiding the list would be our preferred option.

**Comment by hannyprastya** [comment]
Is there any way to hide workspace section?

**Comment by FonduemangVI** [comment]
This is a feature I would greatly appreciate being added. The ability to have the member list or group list hidden from other groups or a more simpler implementation would be to hide them completely from non admins

**Comment by molschimke2** [comment]
I have my first users complaining about this being a GDPR issue.

**Reply by goranskular** [reply]
This is a must have feature, members and groups lists needs to be visible only to admins.

**Comment by wtrucci** [comment]
I agree with @molschimke2. Any news on this front?

**Reply by tommoor** [reply]
`Email address visibility` was added as a security setting a few versions ago

**Reply by wtrucci** [reply]
Hi @tommoor, Thank you for your response, and thank you for all your hard work and for helping to make Outline great. It would be very helpful—especially to make it GDPR-compliant—to have the option to hide users and groups, just as you can with emails. Do you think that would be possible?
Thank you

**Reply by molschimke2** [reply]
I also agree with @wtrucci . For GDPR reasons, we need to hide the user names and personal names. 
As a workaround, we used a firewall setting to prevent users from accessing the page. But that is more like a hack…

**Comment by altroveconsulting** [comment]
Hello! ...I have been following this thread for a few months now, hoping for an option to hide Wiki groups and users from non-admin users. This feature is essential for me as well, so that I can use the platform not just internally, but also as a document management system to keep my clients' documentation updated.

Furthermore, there is a need to restrict non-manager 'writer' users from managing page access permissions. For security reasons, these permissions should only be handled by managers, rather than allowing any writer to view all users/groups and decide who gets access to a page they just created.

As far as I am concerned, not being able to hide the permissions section and the users/groups list from non-manager/non-admin users is a massive limitation. Therefore, I would also like to kindly ask if this could be implemented. It would truly make Outline the absolute best, and above all, allow us to use it in GDPR-compliant environments. Thank you!

**Reply by tommoor** [reply]
This capability is available today in the licensed edition via guest users. Guests are designed specifically for external audiences (like clients, partners, or students) and have a much more restricted view of the workspace.

They can be given access only to specific collections/documents and do not see the full list of workspace members and groups in the same way regular team members do, which helps address the privacy/GDPR concerns you’ve raised.
